src/Security/PetVoter.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Security;
+
+use App\Entity\Pet;
+use App\Entity\User;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authorization\Voter\Voter;
+
+class PetVoter extends Voter
+{
+    //possible actions:
+    const VIEW = 'view';
+    const EDIT = 'edit';
```

Comment:
I don't see view used anywhere, also I would rather see this as a `PetViewVoter` `PetEditVoter`, especially since you don't even need the view one since everyone can see it ...